lib/atlas/preset/scaling.rb

```diff
@@ -17,6 +17,10 @@ class Scaling
       validates :area_attribute, presence: true
       validates :value,          presence: true, numericality: true
       validates :base_value,     presence: true, numericality: true
+
+      def factor
+        value.to_r / base_value.to_r
```

Comment:
It occurs to me that we now have two identical methods which do this; there is also `Atlas::Scaler#scaling_factor`. Could we refactor Scaler to remove that method and instead have it use this new method in `Preset::Scaling#factor`? It already creates a Preset::Scaler so it seems like it might be a good fit...